src/emulation/mod.rs

```diff
@@ -78,14 +78,12 @@ impl SyscallMock {
             }
             (Syscall::Stat, SyscallStop::Exit) if !self.protocol.mocked_files.is_empty() => {
                 let filename = tracee_memory::peek_string(pid, registers.rdi)?;
-                println!(
-                    "mocked files: {:?} - {:?}",
-                    self.protocol.mocked_files.clone().into_iter().map(|f| String::from_utf8(f).unwrap()).collect::<Vec<String>>(),
-                    String::from_utf8(filename.clone())
-                );
                 if self.protocol.mocked_files.contains(&filename) {
-                    println!("huhu");
-                    let mut registers = registers.clone();
+                    if filename.ends_with(b"/") {
```

Comment:
@soenkehahn question: do you think it's ok to start with this as the way to specify to `check-protocols` that you intend for a file path in `foldertree` to be treated as a directory? 

the problem i imagine is, i may accidentally try to mock a directory `/foo/bar/` like this:
```yaml
filetree:
  - /foo/bar
  - /foo/bar/test
```
but if the tests fail, it will be hard to figure out why.

on the other hand it could just be mentioned in the readme, hopefully in a clear enough way that people won't run into this by accident.